manifests/server/conf.pp

```diff
@@ -41,8 +41,9 @@
 # @param syslogfacility Gives the facility code that is used when
 #   logging messages.
 #
-# @param gssapiauthentication  Specifies whether user authentication
-#   based on GSSAPI is allowed.
+# @param gssapiauthentication Specifies whether user authentication
+#   based on GSSAPI is allowed. If the system is connected to an IPA domain,
+#   this will be set to true, based on the existance of the `ipa` fact.
```

Comment:
When the `ipa` fact is present, this parameter will default to `true`, but _can_ be overridden (using APL).